src/Confluent.Kafka/Consumer.cs

```diff
@@ -194,18 +194,37 @@ public void Unsubscribe()
             => consumer.Unsubscribe();
 
         /// <summary>
-        ///     Update the assignment set to \p partitions.
+        ///     Update the assignment set to <param name="partitions" />.
         ///
-        ///     The assignment set is the set of partitions actually being consumed
-        ///     by the KafkaConsumer.
-        /// </summary>
-        /// <remarks>
-        ///     blocks
-        /// </remarks>
+        ///     The assignment set is the complete set of partitions to consume
+        ///     from.
+        /// </summary>
+        /// <param name="partitions">
+        ///     The set of partitions to consume from. If an offset value of
+        ///     -1001 (Offset.Invalid) is specified for a partition, consumption
+        ///     will resume from one after the last committed offset on that
```

Comment:
It will start at the committed offset (the committed offset is the next offset to consume, not the last message consumed)